analyzer-models/src/test/java/ffb/analyzer/models/espn/TeamRecordOverallTests.java

```diff
@@ -0,0 +1,61 @@
+package ffb.analyzer.models.espn;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import ffb.test.utilities.GenericTests;
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Objects;
+
+public class TeamRecordOverallTests {
+    private static final String TEAM_RECORDS = "team-record.json";
+
+    private static ObjectMapper MAPPER;
+
+    @BeforeClass
+    public static void buildPrep() {
```

Comment:
Created #15 - Fully Utilize `DeserializingResourceLoader`. Not worth fixing in this PR.

Reply:
Good idea. Will be best implementing that after both our current pull requests get into the dev branch.